<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 33, 33);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>60</width>
    <height>60</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="10" column="4">
     <widget class="QPushButton" name="btn_paper">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/paper.png</normaloff>images/paper.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="lbl_user">
      <property name="styleSheet">
       <string notr="true">border-color: rgb(255, 255, 127);
border-style: dashed;
border-width: 5px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="5">
     <widget class="QPushButton" name="btn_rock">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/rock.png</normaloff>images/rock.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="lbl_cmptr">
      <property name="styleSheet">
       <string notr="true">border-color: rgb(255, 255, 127);
border-style: dashed;
border-width: 5px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="6">
     <widget class="QPushButton" name="btn_scissors">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/scissors.png</normaloff>images/scissors.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="3">
     <widget class="QLabel" name="lbl_player_score">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 11pt &quot;Arial Black&quot;;
color: rgb(85, 255, 0);</string>
      </property>
      <property name="text">
       <string>Player : 0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="3">
     <widget class="QLabel" name="lbl_computer_score">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 11pt &quot;Arial Black&quot;;
color: rgb(85, 255, 0);</string>
      </property>
      <property name="text">
       <string>Computer : 0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="3">
     <widget class="QLabel" name="lbl_win">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);
font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
